scripts/entry/app.py

```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass
+from typing import Dict
+import cfn_flip
+import json
+import subprocess
+import pathlib
+
+class Utils:
+    @staticmethod
+    def get_dict_from_template(filepath):
+        with open(filepath) as f:
+            template_raw = f.read()
+        template_flat_json = cfn_flip.to_json(template_raw)
+        template_dict = json.loads(template_flat_json)
+        return template_dict
+
+@dataclass
+class EntryTemplateFactory:
+    boilerplate: Dict
+    main:        Dict
+    entry:       Dict = None
+
+
+    @staticmethod
+    def new(**kwargs):
```

Comment:
Is there a reason for this to take `**kwargs` instead of the required `main_filepath` and `boilerplate_filepath` parameters?

Reply:
I see you have a check for `boilerplate_filepath`, but that can still be presented as an optional param with default `None`
```python
    def new(main_filepath: str, boilerplate_filepath: Optional[str] = None):
```

or, based on your logic below, maybe just default it to `"input/boilerplate.yaml"`?
```python
    def new(main_filepath: str, boilerplate_filepath: str = "input/boilerplate.yaml"):
```

Reply:
Also would prefer to see `pathlib.Path` used in place of `str` for filepath arguments